$NetBSD: patch-ab,v 1.4 2015/03/29 15:28:43 rodent Exp $

Don't build with debugging symbols by default nor hardcode compiler flags.

--- qmake.inc.in.orig	2012-03-23 06:10:54.000000000 +0000
+++ qmake.inc.in
@@ -17,7 +17,7 @@ CPPUNIT_LIBS = @CPPUNIT_LIBS@
 
 # CONFIG -= nostrip
 # CONFIG += release
-CONFIG += debug
+#CONFIG += debug
 
 unix {
     PREFIX = @PREFIX@
@@ -36,11 +36,10 @@ unix {
     OBJECTS_DIR = .obj
 
     QMAKE_CFLAGS_DEBUG += -Wno-unused-parameter
-    QMAKE_CFLAGS_RELEASE += -Wno-unused-parameter
+    QMAKE_CFLAGS_RELEASE += @CXXFLAGS@
     QMAKE_CXXFLAGS_DEBUG += -Wno-unused-parameter
-    QMAKE_CXXFLAGS_RELEASE += -Wno-unused-parameter
+    QMAKE_CXXFLAGS_RELEASE += @CXXFLAGS@
 
-    !macx {
 
         exec_prefix = @EXEC_PREFIX@
         DESTDIR =
@@ -68,7 +67,6 @@ unix {
         CONFIG += warn_on
 
 #		DEFINES += __STDC_FORMAT_MACROS
-    }
 }
 
 win32 {
